packages/hardcaml-reedsolomon/hardcaml-reedsolomon.0.1/files/hardcaml-reedsolomon.install

```diff
@@ -0,0 +1,4 @@
+bin: [
+"hcrsencode.native" {"hardcaml-rsencode"}
+"hcrsdecode.native" {"hardcaml-rsdecode"}
+]
```

Comment:
You can put these directly in your source repo, btw (so it works with local `opam pin` workflows as well)